features/projects/types/project.types.ts

```diff
@@ -5,9 +5,9 @@ export enum ProjectLanguage {
 }
 
 export enum ProjectStatus {
-  stable = "stable",
+  info = "info",
   warning = "warning",
-  critical = "critical",
+  error = "error",
```

Comment:
Great job detecting the problem correctly! That's the main challenge in this task. I've seen complicated solutions here, so honestly 👍

Reply:
This is actually very difficult!! haha! but it in the end, I thought this is the better solution.